src/constants/initialState.js

```diff
@@ -7,6 +7,7 @@ const INITIAL_STATE = {
     { id: uuidv4(), title: 'Tales from Andromeda', category: 'Sci-fi' },
```

Comment:
- [ ] Category for Tales from Andromeda should be `Sci-Fi` instead `Sci-fi`, just like is defined in the category constants.

- [ ] Adventure it's not a defined category.
https://github.com/praz99/bookstore/blob/89247f29c5df63d45d86626cc13b45f4fa2db356/src/constants/initialState.js#L6